packages/common-all/src/analytics.ts

```diff
@@ -116,6 +116,8 @@ export enum ConfigEvents {
   DeprecatedConfigMessageConfirm = "DeprecatedConfigMessageConfirm",
   DeprecatedConfigMessageShow = "ShowDeprecatedConfigMessage",
   MissingDefaultConfigMessageConfirm = "MissingDefaultConfigMessageConfirm",
+  DuplicateConfigMessageShow = "DuplicateConfigMessageShow",
```

Comment:
Nit: can we amend this to be "DuplicateConfigEntryMessageShow"?

To avoid confusing this to mean that the user has two `dendron.yml` files for example.